core/services/pipeline/models.go

```diff
@@ -39,13 +40,18 @@ type Run struct {
 	Meta           JSONSerializable `json:"meta"`
 	// The errors are only ever strings
 	// DB example: [null, null, "my error"]
-	Errors RunErrors `json:"errors" gorm:"type:jsonb"`
+	Errors RunErrors        `json:"errors" gorm:"type:jsonb"`
+	Inputs JSONSerializable `json:"inputs" gorm:"type:jsonb"`
```

Comment:
It's the initial inputs (+ meta) for the run. Certain jobs have initial inputs. We persist these because they need to still be present in the `Vars` env after a resume.

Reply:
Gotcha, makes sense. Can it be added to the state machine check?